.github/workflows/deploy-lambda.yml

```diff
@@ -10,11 +10,17 @@ jobs:
 
     steps:
     - uses: actions/checkout@master
-    - name: Deploy code to Lambda
-      uses: mariamrf/py-lambda-action@v1.0.1
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v2
       with:
-        lambda_layer_arn: 'arn:aws:lambda:us-east-1:196072577018:function:ralph'
-        lambda_function_name: 'lambda_function.lambda_handler'
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-2
```

Comment:
Couldn't we use `{{ env.REGION }}` here? Even if not, we'd want us-east-1 instead